React/Modules/RCTUIManager.h

```diff
@@ -148,8 +148,8 @@ RCT_EXTERN NSString *const RCTUIManagerRootViewKey;
 - (void)rootViewForReactTag:(NSNumber *)reactTag withCompletion:(void (^)(UIView *view))completion;
 
 /**
- * Finds a view that is tagged with {@param nativeId} as its nativeID prop
```

Comment:
The original is giving a "documentation error"  when compiling and running the unit and integration tests.